src/components/MuteButton/MuteButton.js

```diff
@@ -0,0 +1,25 @@
+/**
```

Comment:
NOTE: I had a similar approach but placing the button in _src/components/Participants/Actions/Mute.js_ But that kind of (re)organization can be done later. In fact, I prefer to do it later.